README.md

```diff
@@ -21,17 +21,17 @@ Since the container needs to be privileged, add the reqired RBAC rules:
 oc create -f backup-rbac.yaml
 ```
 
-Then adjust storage to your needs in `backup-storage.yaml` and deploy it. The example uses NFS but you can use any storage class you want.
+Then adjust storage to your needs in `backup-storage.yaml` and deploy it. The example uses NFS but you can use any storage class you want (`hostPath` or `provioning`):
```

Comment:
provioning?

Reply:
yeah provisioning meaning the auto provisioning setup of your cluster, is that unclear?

Reply:
@paraenggu The suggestions feature is the button with the ± sign on it.
```suggestion
Then adjust the storage configuration to your needs in `backup-storage.yaml` and deploy it. The example uses NFS but you can use any storage class you want (`hostPath` or `provisioning`):
```

Reply:
> yeah provisioning meaning the auto provisioning setup of your cluster, is that unclear?

The backup-storage.yaml refers to a pre-created PV/PVC, on should probably never use a dynamic provisioner to create a PV, as you need to make sure that you can access the data outside of a running cluster (in case it failed an you have to restore it). Therefor you would have to make sure, that the PV details are known in advance and won't change later on.